modules/ai-initial-oci-resources-services.adoc

```diff
@@ -2,18 +2,19 @@
 //
 // * installing/installing_oci/installing-oci-assisted-installer.adoc
 
-:_mod-docs-content-type: PROCEDURE
-[id="creating-oci-resources-services_{context}"]
-= Creating OCI resources and services
+:_mod-docs-content-type: CONCEPT
+[id="ai-initial-oci-resources-services_{context}"]
+= Initial OCI resources and services
 
-Create {oci-first} resources and services so that you can establish infrastructure with governance standards that meets your organization's requirements.
+You must create {oci-first} resources and services so that you can establish infrastructure with governance standards that meets your organization's requirements.
 
-By creating a compartment, you can better organize, restrict access, and set usage limits to {oci} resources. Additionally, by creating an object storage bucket, you can safely and securely store the ISO image. You can access the image at a later stage for the purposes of booting the instances, so that you can then create your cluster.
+Before you create {oci} resources and services, you must meet certain prerequisites that require you to configure an {oci} account that manages {oci} resources and services, and hosts the {product-title} cluster.
 
-.Prerequisites
+After you meet these prerequisites, you must download an archive file from an Oracle resource. The archive file includes files for creating cluster resources and custom manifests, and a script that when run creates {oci} resources, such as DNS records, an instance, and so on.
```

Comment:
Flow for this is different than our [original docs](https://docs.openshift.com/container-platform/4.15/installing/installing_oci/installing-oci-assisted-installer.html#downloading-manifest-files-resources-oci_installing-oci-assisted-installer). I asked Adrien Gentil, our RH engineer.